app/models/concerns/type_of_employment.rb

```diff
@@ -7,6 +7,10 @@ def not_working?
     income.employment_status.include?(EmploymentStatus::NOT_WORKING.to_s)
   end
 
+  def employed?
+    income.employment_status.include?(EmploymentStatus::EMPLOYED.to_s)
+  end
```

Comment:
I think we should make this EMPLOYED only. As if employement_status = ['employed', 'self-employed'] I think the route and therefore validations is unique

Reply:
Decided to leave it for now and update when we get around to self employed journey